### DISCUSSION TITLE
20220627(월)

### DISCUSSION THREAD

**Discussion by YunDongHwan** [innovationacademy-kr/42dashboard#65] [Q&A]
## 질문 목록
1. @injectable()을 export가 있는데에도 사용하는 이유는???
2. @injectable(Employ) 처럼 넣어주는 이유는?

**Comment by Likilee** [comment]
아래 쓰레드 답변이 도움이 될거 같아 공유드립니다 :)

https://stackoverflow.com/questions/64349628/what-is-injectable-in-nestjs?answertab=trending#tab-top
